test/E2ETests/E2ETests/Storage/QueueEndToEndTests.cs

```diff
@@ -166,60 +166,5 @@ public async Task QueueOutput_PocoList_Succeeds()
             IEnumerable<string> queueMessages = await StorageHelpers.ReadMessagesFromQueue(Constants.Queue.OutputBindingNamePOCO);
             Assert.True(queueMessages.All(msg => msg.Contains(expectedQueueMessage)));
         }
-
-        [Fact]
```

Comment:
Are these tests being put elsewhere?

Reply:
Yeah they were moved to the BlobEndToEndTests file